include/kwk/settings/axis.hpp

```diff
@@ -142,4 +155,24 @@ namespace kwk
 
   /// Predefined axis for channel
   inline constexpr auto channel = axis<"channel">;
+
+  // thin as_dimension
+         constexpr auto extent(std::integral             auto const dim) noexcept { return dim; }
```

Comment:
why the duplication from as_dimension. Again, we tried ot get to a regular, straight forward setup before.
What are the rationale for these changes cause from afar it feels like ad hoc fixes to issue our use cases never encoutnered so I would appreciate a bit of backstory.

Reply:
Ping me on discord so we can setup a date for this live sync :D 
I am sure your doing the thing that needs to be done but from afar it is complicated to get the big picture.

Reply:
The CPO system was meant to prevent issues I had in other options handling where specialisation were not found correctly.
Now, if the specialisation on those functions are not really needed becasue nobody will add new extent, we can shrink back to specific functions.